superset/common/query_context.py

```diff
@@ -129,15 +129,14 @@ def get_data(  # pylint: disable=invalid-name,no-self-use
     def get_single_payload(self, query_obj: QueryObject) -> Dict[str, Any]:
         """Returns a payload of metadata and data"""
         payload = self.get_df_payload(query_obj)
-        df = payload.get("df")
-        status = payload.get("status")
+        df = payload["df"]
```

Comment:
Note the `get` is not required as `get_df_payload` is guaranteed to return a dictionary with the `df` and `status` keys per [here](https://github.com/apache/incubator-superset/blob/5b690f94116b5b77c0af566d24f298cc6db469b9/superset/common/query_context.py#L234).